internal/receiver/discoveryreceiver/endpoint_tracker.go

```diff
@@ -82,6 +93,26 @@ func (et *endpointTracker) start() {
 		go observable.ListAndWatch(n)
 	}
 	et.correlations.Start()
+	go et.startEmitLoop()
+}
+
+func (et *endpointTracker) startEmitLoop() {
+	go func() {
```

Comment:
I don't think this function should start its own goroutine, since a goroutine is already started when calling this method, right?

Reply:
good catch! 👍